keyboards/edc40/info.json

```diff
@@ -0,0 +1,10 @@
+{
+    "keyboard_name": "edc40",
+    "url": "",
+    "maintainer": "OJ",
+    "width": 12,
+    "height": 4,
+    "layouts": {
+        "LAYOUT": [{"label":"Esc", "x":0, "y":0}, {"label":"Q", "x":1, "y":0}, {"label":"W", "x":2, "y":0}, {"label":"E", "x":3, "y":0}, {"label":"R", "x":4, "y":0}, {"label":"T", "x":5, "y":0}, {"label":"Y", "x":6, "y":0}, {"label":"U", "x":7, "y":0}, {"label":"I", "x":8, "y":0}, {"label":"O", "x":9, "y":0}, {"label":"P", "x":10, "y":0}, {"label":"Backspace", "x":11, "y":0}, {"label":"Tab", "x":0, "y":1}, {"label":"A", "x":1.25, "y":1}, {"label":"S", "x":2.25}, {"label":"D", "x":3.25, "y":1}, {"label":"F", "x":4.25, "y":1}, {"label":"G", "x":5.25, "y":1}, {"label":"H", "x":6.25, "y":1}, {"label":"J", "x":7.25, "y":1}, {"label":"K", "x":8.25, "y":1}, {"label":"L", "x":9.25, "y":1}, {"label":"Enter", "x":10.25, "y":1}, {"label":"Shift", "x":0, "y":2}, {"label":"Z", "x":1.75, "y":2}, {"label":"X", "x":2.75, "y":2}, {"label":"C", "x":3.75, "y":2}, {"label":"V", "x":4.75, "y":2}, {"label":"B", "x":5.75, "y":2}, {"label":"N", "x":6.75, "y":2}, {"label":"M", "x":7.75, "y":2}, {"label":",", "x":8.75, "y":2}, {"label":"RShift", "x":9.75, "y":2}, {"label":"LCtl", "x":0, "y":3}, {"label":"LGui", "x":1.25, "y":3}, {"label":"", "x":4.75, "y":3}, {"label":"RAlt", "x":7.75, "y":3}, {"label":"RCtl", "x":9, "y":3}]
+    }
+}
```

Comment:
```suggestion
{
    "keyboard_name": "edc40",
    "url": "",
    "maintainer": "OJ",
    "width": 12,
    "height": 4,
    "layouts": {
        "LAYOUT": {
            "layout": [
                {"x": 0, "y": 0},
                {"x": 1, "y": 0},
                {"x": 2, "y": 0},
                {"x": 3, "y": 0},
                {"x": 4, "y": 0},
                {"x": 5, "y": 0},
                {"x": 6, "y": 0},
                {"x": 7, "y": 0},
                {"x": 8, "y": 0},
                {"x": 9, "y": 0},
                {"x": 10, "y": 0},
                {"x": 11, "y": 0},

                {"x": 0, "y": 1},
                {"x": 1.25, "y": 1},
                {"x": 2.25, "y": 1},
                {"x": 3.25, "y": 1},
                {"x": 4.25, "y": 1},
                {"x": 5.25, "y": 1},
                {"x": 6.25, "y": 1},
                {"x": 7.25, "y": 1},
                {"x": 8.25, "y": 1},
                {"x": 9.25, "y": 1},
                {"x": 10.25, "y": 1},

                {"x": 0, "y": 2},
                {"x": 1.75, "y": 2},
                {"x": 2.75, "y": 2},
                {"x": 3.75, "y": 2},
                {"x": 4.75, "y": 2},
                {"x": 5.75, "y": 2},
                {"x": 6.75, "y": 2},
                {"x": 7.75, "y": 2},
                {"x": 8.75, "y": 2},
                {"x": 9.75, "y": 2},

                {"x": 0, "y": 3},
                {"x": 1.25, "y": 3},
                {"x": 4.75, "y": 3},
                {"x": 7.75, "y": 3},
                {"x": 9, "y": 3}
            ]
        }
    }
}
```
Just noticed the lint check wasn't happy about this.
Is this correct? (you can preview it in Configurator by pressing Ctrl+Shift+I)
![image](https://user-images.githubusercontent.com/4781841/100398320-2182fd80-30a2-11eb-9e92-10632da61a46.png)

Reply:
hmm, spacing seems off, I need to fix the info.json file, I assume that's what the configurator pulls from

Reply:
now it's all good